config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanics", to: "mechanics#show"
 end
```

Comment:
Because we are going to a show page for a specific mechanic, we want our route to be able to take in the ID of that mechanic so that we can use that information in our controller. The route should look like this:

```ruby
get "/mechanics/:id", to: "mechanics#show"
```

If we wanted to get ALL of the mechanics then the route would look like this:

```ruby
get "/mechanics", to: "mechanics#index"
```